test/e2e/common.go

```diff
@@ -581,6 +582,7 @@ func MachineToVMName(ctx context.Context, cli client.Client, m *clusterv1.Machin
 	return "", fmt.Errorf("no matching Metal3Machine found for current Machine")
 }
 
+// Gets IPAddress based on machine, from machine -> m3machine -> m3data -> IPAddress.
```

Comment:
Sorry for being nit picky. Could you make all of these comments follow the go doc standard so that they begin with the name of the function? For example this one should be
```suggestion
// MachineToIPAddress gets IPAddress based on machine, from machine -> m3machine -> m3data -> IPAddress.
```

I know it may seem redundant, but this way all kinds of developer tools or documentation generators can pick them up and make nice things happen :slightly_smiling_face: 
If you want to know more about go doc comments, check this nice page: https://go.dev/doc/comment

Reply:
Of course, I just hadn't realized that was the format :)